erpnext/agriculture/doctype/destroyed_plant_log/destroyed_plant_log.js

```diff
@@ -2,7 +2,16 @@
 // For license information, please see license.txt
 
 frappe.ui.form.on('Destroyed Plant Log', {
-	// refresh: function(frm) {
-
-	// }
+	category: function(frm){
+		frm.toggle_display('plant', frm.doc.category == "Plant");
+		if (frm.doc.category == "Plant"){
+			frm.set_df_property('plant', 'reqd', 1);
+			frm.set_df_property('plant_batch', 'reqd', 0);
+		}
+		frm.toggle_display('plant_batch', frm.doc.category == "Plant Batch");
```

Comment:
```suggestion
		frm.toggle_display('plant_batch', frm.doc.category === "Plant Batch");
```

Reply:
![image](https://user-images.githubusercontent.com/56826544/95584234-fb09f400-0a5a-11eb-97a9-9bbe1ffe9e62.png)

Reply:
-  Use triple equal to in JS.